migrations/1597956943277-lernstore_view_permission.js

```diff
@@ -0,0 +1,46 @@
+const { connect, close } = require('../src/utils/database');
+const Roles = require('../src/services/role/model');
+
+module.exports = {
+	up: async function up() {
```

Comment:
I would wrap everything which is inside an async function in a try-catch to handle errors.

Reply:
should be done from the template